_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
@jeremiak I agree that ideally we wouldn't need this. However, I've seen other pattern libraries that use `*-library` or `*-styleguide` to make this distinction, so I don't think it's true for every case. I would vote move forward bc it's crunch time and we can open an issue to change this when we have more time. 

Also, note: we haven't done v2's of the footer yet so this *could* help.